File: Practice/AllDomains/Tutorials/30DaysOfCode/Day26NestedLogic.cs
/*	author:@shivkrthakur  */
using System;
using System.Collections.Generic;
using System.IO;
class Solution {
    static void Main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
            int[] userReturnDate = Array.ConvertAll(Console.ReadLine().Split(' '), Int32.Parse);
            int[] actualReturnDate = Array.ConvertAll(Console.ReadLine().Split(' '), Int32.Parse);

            if (userReturnDate[2] == actualReturnDate[2])
            {
                if (userReturnDate[1] == actualReturnDate[1])
                {
                    if (userReturnDate[0] > actualReturnDate[0])
                    {
                        int days = userReturnDate[0] - actualReturnDate[0];
                        Console.WriteLine(15 * days);
                        return;
                    }
                }
                else if (userReturnDate[1] > actualReturnDate[1])
                {
                    int monthsLate = userReturnDate[1] - actualReturnDate[1];
                    Console.WriteLine(500 * monthsLate);
                    return;
                }
            }
            else if (userReturnDate[2] > actualReturnDate[2])
            {
                Console.WriteLine("10000");
                return;
            }
            Console.WriteLine("0");
    }
}

File: Practice/AllDomains/Tutorials/30DaysOfCode/Day26NestedLogic.js
/*	author:@shivkrthakur  */
function processData(input) {
    //Enter your code here
    input = input.split('\n');
    var aD = input[0].split(' ').map(Number);
    var eD = input[1].split(' ').map(Number);

    var ad = aD[0];
    var am = aD[1];
    var ay = aD[2];

    var ed = eD[0];
    var em = eD[1];
    var ey = eD[2];

    var fine = 0;
    if(ad > ed && am <= em && ay == ey){
        fine += 15 * (ad - ed);
    }
    else if(am > em && ay == ey){
        fine += 500 * (am - em);
    }
    else if(ay > ey){
        fine += 10000;
    }
    console.log(fine);
}

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});
